src/script/sign.h

```diff
@@ -32,6 +32,16 @@ class BaseSignatureCreator {
     virtual bool CreateSig(const SigningProvider& provider, std::vector<unsigned char>& vchSig, const CKeyID& keyid, const CScript& scriptCode, SigVersion sigversion) const =0;
 };
 
+class SimpleSignatureCreator : public BaseSignatureCreator
```

Comment:
comments about the purpose of this would be nice

Reply:
Added a few words to describe what it is. :)